src/style.css

```diff
@@ -1 +1,63 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  margin: 0;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: Avenir;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  margin-top: 20px;
+  margin-left: 8px;
+  margin-right: 8px;
```

Comment:
You are adding right margin to the form. This is not necessary unless there is a specific requirement to do so. Let the content size dictate it to avoid overflow or accidental scroll bar.